qa/suites/rbd/nvmeof/base/install.yaml

```diff
@@ -0,0 +1,37 @@
+use_shaman: True
+tasks:
+- cephadm:
+- cephadm.shell:
+    host.a:
+    # get state before nvmeof deployment
+    - ceph orch status
+    - ceph orch ps
+    - ceph orch ls
+    - ceph orch host ls
+    - ceph orch device ls
+    - ceph osd lspools
+    # create pool
+    - ceph osd pool create mypool
+    - rbd pool init -p mypool
+    - ceph osd pool ls detail
+    # deploy nvmeof
+    ## Uncomment to test specific nvmeof images
+    ## - ceph config set mgr mgr/cephadm/container_image_nvmeof quay.io/ceph/nvmeof:latest
+    - ceph orch apply nvmeof mypool --placement="1 $(hostname)"
+    - ceph orch ps --refresh
+    - ceph versions
+    - ceph -s
+    - ceph orch ls
+    - rbd create mypool/myimage --size 8Gi
```

Comment:
It's because they run on different remotes - the RBD image is created on `host.a` and workunit script runs on `client.1`. 
To ensure we don't have to keep these things synced between two files, I can take image name and pool name as part of `cephadm.nvmeof_gateway_cfg` config and add them to `/etc/ceph/nvmeof.env` so they can be later exported in the shell script.

Reply:
> It's because they run on different remotes - the RBD image is created on `host.a` and workunit script runs on `client.1`. 

It's the same Ceph cluster, isn't it?  You should be able to run RBD commands from any remote.

> To ensure we don't have to keep these things synced between two files, I can take image name and pool name as part of `cephadm.nvmeof_gateway_cfg` config and add them to `/etc/ceph/nvmeof.env` so they can be later exported in the shell script.

That would be wrong because RBD image details have nothing to do with the gateway configuration.